### PR TITLE
add simple test that katello-change-hostname works

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
Does it make sense to add some randomness to the new hostname so you can actually run it multiple times?

File: bats/fb-test-katello-change-hostname.bats [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+@test "change hostname to katello.example.com" {
+  katello-change-hostname --username admin --password changeme --assumeyes katello.example.com
+}
```
[the remark this answers is not in the document]
Comment (evgeni) [reply bats/fb-test-katello-change-hostname.bats:8]: so something like:

```bash
NEW_HOSTNAME="katello-$$.example.com"
katello-change-hostname … $NEW_HOSTNAME
[ "$(hostname -f) = "${NEW_HOSTNAME}" ]
```

[the remark this answers is not in the document]
Comment (ekohl) [reply bats/fb-test-katello-change-hostname.bats:8]: Pretty much yes.

[the remark this answers is not in the document]
Comment (evgeni) [reply bats/fb-test-katello-change-hostname.bats:10]: done

**Comment by evgeni** [comment]
This will break on Satellite as there we *have* to call `satellite-change-hostname` (it validates the "right" scenario).

**Pull request merged**